utils/bump_version.py

```diff
@@ -42,7 +56,7 @@ def bump_version(path, version_info, in_develop=True) -> None:
         f.write('\n'.join(lines) + '\n')
 
 
-def parse_version(version):
+def parse_version(version: str) -> VersionInfo:
```

Comment:
The construction normalizes the output so that it has 5 elements, so yes, let's remove the union in the type.